seedfarmer/cli_groups/_bundle_group.py

```diff
@@ -77,21 +79,26 @@ def bundle() -> None:
     help="Full path of the bundle object in SeedKit bucket",
     required=True,
 )
+@click.option(
+    "--region",
+    default=None,
+    help="The AWS region used to create a session",
+    required=False,
+)
 def store_bundle(
-    deployment: str,
-    group: str,
-    module: str,
-    bucket: str,
-    origin: str,
+    deployment: str, group: str, module: str, bucket: str, origin: str, region: Optional[str] = None
 ) -> None:
     _load_project()
     print(f"{deployment} - {group} - {module} -{bucket} - {origin}")
+    SessionManager.bind(SessionManagerLocalImpl())
+    session = (
+        SessionManager()
+        .get_or_create(region_name=region)
+        .get_deployment_session(account_id="000000000000", region_name=str(region))
+    )
```

Comment:
just confirming we only bind local impl because bundle and meta commands are only meant to be executed in the "current" account by seed-farmer itself?

Reply:
Exactly.  These should only be executed by SeedFarmer when running in Codebuild, so it technically has to be the SessionLocal.  I added the region flag just to be sure that we write to the proper region.